.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,49 @@
+name: 2. ⭐ Feature Request
+description: Suggest an idea for a new function in c:geo directly to our GitHub tracker 
+# title: ""
+labels: [Feature Request, Unverified]
+# assignees: 
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to submit your idea!
+  - type: textarea
+    id: summary
+    attributes:
+      label: Tell us your idea!
+      description: A clear and detailed description of the function you are missing/would like to see in c:geo.
+      placeholder: "
+        Examples:\n 
+        There is no possibility to...\n
+        I'm always frustrated when...\n
+        It would be easier to use if
+        "
+    validations:
+      required: true
+  - type: textarea
+    id: solution
+    attributes:
+      label: What solution would you suggest?
+      description: Describe your sugggested solution and the way it should be implemented.
+      placeholder: "
+      Examples:\n
+      There should be a new menu item xyz in menu abc, which performs [...]\n
+      Change the existing function to include [...]
+      "
+    validations:
+      required: true
```

Comment:
Really required? Maybe same as for bugs...

Some things can  be explained in one sentence, other things need further explanation... 

I don't think it makes sense to technically force users to enter something. They can still enter a "-" which doesn't help us either. I'm quite sure that users will use the text box if it makes sense, independent of whether it is required or not.

Reply:
OK, I will set it to false.

Reply:
Done